orchagent/portsorch.cpp

```diff
@@ -3095,6 +3095,38 @@ bool PortsOrch::removeVlanHostIntf(Port vl)
     return true;
 }
 
+void PortsOrch::updateDbPortFlapCount(Port& port, sai_port_oper_status_t pstatus)
+{
+    SWSS_LOG_ENTER();
+
+    ++port.m_flap_count;
+    vector<FieldValueTuple> tuples;
+    FieldValueTuple tuple("flap_count", std::to_string(port.m_flap_count));
+    tuples.push_back(tuple);
+    m_portTable->set(port.m_alias, tuples);
```

Comment:
@prgeor should be at the end of the function: gather field/value tuples and then set at once. No need to have duplicate calls to DB